File: agent/entity/performance.go
package entity

// Performance 实时性能指标
type Performance struct {
	CPUPercent    float64 `json:"cpu_percent"`    // CPU使用率
	MemPercent    float64 `json:"mem_percent"`    // 内存使用率
	DiskPercent   float64 `json:"disk_percent"`   // 磁盘使用率
	TCPConnection int     `json:"tcp_connection"` // TCP连接数量
	NetworkRate   float64 `json:"network_rate"`   // 网络速率，KB/S
}

File: server/src/main/java/org/pi/server/model/dto/Performance.java
package org.pi.server.model.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Performance {
    private String cpuPercent;
    private String memPercent;
    private String diskPercent;
    private String tcpConnection;
    private String networkRate;
    private LocalDateTime time;
    private String agentID;
}
